testdata/config/sign-job/sign-job.yaml

```diff
@@ -2,15 +2,16 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: check-oidc
+  name: sign-job
 spec:
   template:
     spec:
       restartPolicy: Never
       automountServiceAccountToken: false
       containers:
-      - name: check-oidc
-        image: gcr.io/projectsigstore/cosign/ci/cosign@sha256:57be112e1d5686ccf6a5a78910fcfff648d3a93c4bca10ccc0c2304dfe1492d7
+      - name: cosign
+        # Built from ci on 2022-03-15
+        image: gcr.io/projectsigstore/cosign/ci/cosign@sha256:8f7f1a0e7cef67c352f00acd14791d977faa8d1cd47a69f9c880a5185c44ffbb
```

Comment:
why not use the released one? or this is something that is not released yet?

Reply:
Yeah, I need this one:
https://github.com/sigstore/cosign/pull/1610

I'll create an issue to track that we switch after the next release of cosign.

Reply:
thanks for the clarification
maybe we can add a TODO
(non-blocking)